driver/modern_bpf/helpers/base/push_data.h

```diff
@@ -214,7 +214,8 @@ static __always_inline void push__previous_character(uint8_t *data, uint64_t *pa
  * @param charbuf_pointer pointer to the charbuf.
  * @param limit maximum number of bytes that we read in case we don't find a `\0`
  * @param mem tell where it must read: user-space or kernel-space.
- * @return (uint16_t) the number of bytes written in the buffer. Could be '0' if the passed pointer is not valid.
+ * @return (uint16_t) the number of bytes written in the buffer. Returns '0' if the passed pointer is not valid.
+ *         Returns `1` if the provided pointer points to an empty string "".
```

Comment:
before this patch we returned `0` and so we sent a param with len 0 to userspace